src/store/modules/wallet.js

```diff
@@ -413,6 +425,119 @@ const actions = {
     }
   },
 
+  async fetchFolloweeWalletEvent({ state, commit, dispatch }) {
```

Comment:
Can we reuse this function/store in notification page?

Reply:
Sure, but I prefer to do it in another PR 🌚.